ui/src/app/base/components/Stepper/_index.scss

```diff
@@ -0,0 +1,41 @@
+@mixin Stepper {
+  .stepper {
```

Comment:
I'm never quite sure if this should be `p-stepper`...

Reply:
Yeah, I guess we never really codified anything. I've tended toward not using `.p-whatever` just because I don't want MAAS components to be misconstrued as Vanilla components, but it's never actually been an issue. 🤷🏽‍♂️